Documentation/aws.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Running Rook in AWS
+weight: 25
+indent: true
+---
+
+# Running Rook in AWS
+
+Rook can functionally replace Elastic Block Storage (EBS), Elastic File System (EFS) and S3 with Rook ObjectStore. Potentially reducing storage cost, increasing performance and portability of your Kubernetes workloads.
+
+:information_source:  
+Please refer to [Creating Rook Clusters][3] on how to customize your cluster to use specific devices.
+
+* To deliver robust performance at reasonable cost, consider using instances backed by [*instance store*][1] for Rook cluster nodes. Utilization of storage devices directly attached to the instance will reduce latency and provide more consistent IO performance.
+* The ephemeral nature of the *instance store* is handled by resiliency build into Rook. Make sure to configure `replicas` parameter in accordance with your durability requirements.
+* Multiple EBS devices can be [striped into RAID array][2] and used as Rook devices to improve performance of low cost EBS volumes.
```

Comment:
What is the advantage of striping EBS devices over instance store? I thought instance store was the best option

Reply:
EBS is persistant, instance store is ephemeral, EBS devices can be attached to existing instances of any type, by striping cheap EBS volumes performance is increased, this type of volumes can be of any size vs. instance store.

There are scenarios when striping EBS could be preferred to instance store, it just another option.